clients/lsp-angular.el

```diff
@@ -75,13 +75,13 @@ Has no effects when `lsp-clients-angular-language-server-command' is set."
          ;; so we "cache" its results after running once
          (setq lsp-clients-angular-language-server-command
                (list
-                "node"
-                (f-join node-modules-path "@angular/language-server")
-                "--ngProbeLocations"
-                node-modules-path
+                "ngserver"
+                "--stdio"
                 "--tsProbeLocations"
                 node-modules-path
-                "--stdio"))
+                "--ngProbeLocations"
+                (f-join node-modules-path "@angular/language-server/node_modules/")
+                ))
```

Comment:
Can you avoid having trailing `))`? Thanks!

Reply:
Well, If I understand correctly, I changed trailing `))`

Reply:
I meant put it on the same line:

```diff
+ (f-join node-modules-path "@angular/language-server/node_modules/")))
- (f-join node-modules-path "@angular/language-server/node_modules/")
-                 ))
```

Reply:
My bad, now it is applied